core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.scss

```diff
@@ -0,0 +1,116 @@
+@use "variables" as *;
+@import "dotcms-theme/utils/theme-variables";
+@import "dotcms-theme/utils/extends";
+
+.dotDeviceSelector__panel {
+    display: flex;
```

Comment:
If the columns are all the same width is safer to use css grids because flex might change based on the content size.